assets/railcraft/lang/ru_RU.lang

```diff
@@ -394,44 +473,30 @@ railcraft.gui.help=Помощь
 
 railcraft.gui.ic2.energy.rate=Коэффициент: %s EU
 
-railcraft.gui.item.loader.all.tip=Переместить все подходящие предметы
-railcraft.gui.item.loader.all=Все
-railcraft.gui.item.loader.buffer=Буфер
-railcraft.gui.item.loader.complete.tip=Сигнал крас. камня по завершению
-railcraft.gui.item.loader.complete=Полностью
-railcraft.gui.item.loader.excess.tip=Оставить предметы из фильтра,\n переместить остальное
-railcraft.gui.item.loader.excess=Такие же
-railcraft.gui.item.loader.partial.tip=Сигнал крас. камня, если предметы не были перемещены\n и вагонетка не пустая
-railcraft.gui.item.loader.partial=Частично
-railcraft.gui.item.loader.immediate.tip=Сигнал крас. камня,\n если предметы не были перемещены
-railcraft.gui.item.loader.immediate=Немедленно
-railcraft.gui.item.loader.manual.tip=Никогда не излучать сигнал крас. камня
-railcraft.gui.item.loader.manual=Вручную
-railcraft.gui.item.loader.stock.tip=Перемещать предметы\n до совпадения с фильтром
-railcraft.gui.item.loader.stock=Содержимое
-railcraft.gui.item.loader.transfer.tip=Переместить в точности столько предметов,\n сколько их в фильтре
-railcraft.gui.item.loader.transfer=Количество
+gui.railcraft.manipulator.buffer=Буфер
+gui.railcraft.manipulator.redstone.complete.name=Завершено
+gui.railcraft.manipulator.redstone.complete.tips=Продолжать до конца операции
+gui.railcraft.manipulator.redstone.immediate.name=Immediate
```

Comment:
Can you please explaiun what exact is Charge Trap? 715 tile.railcraft.charge.trap.name=Charge Trap